pkg/handlers/delete.go

```diff
@@ -3,23 +3,23 @@ package handlers
 import (
 	"net/http"
 
-	"github.com/Ubivius/microservice-template/pkg/data"
+	"github.com/Ubivius/microservice-matchmaking/pkg/data"
 )
 
-// Delete a product with specified id from the database
-func (productHandler *ProductsHandler) Delete(responseWriter http.ResponseWriter, request *http.Request) {
-	id := getProductID(request)
-	productHandler.logger.Println("Handle DELETE product", id)
+// Delete a player with specified id from the queue
+func (queueHandler *QueueHandler) Delete(responseWriter http.ResponseWriter, request *http.Request) {
```

Comment:
Is the goal to be able to remove a player from the queue if he desists from the client? If so, that's ok. If not then we just have this function because DELETE was part of a task title

Reply:
Yep was the plan